### PR TITLE
feat: add react-select

### PR DESCRIPTION
**Pull request by raop155**
# Description

- Use the `react-select` module and replace all select/combo box components to match UI/UX

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +37,7 @@
 		"react-i18next": "12.1.5",
 		"react-query": "3.39.3",
 		"react-router-dom": "6.7.0",
+		"react-select": "^5.7.2",
```
Comment (valentinoConti) [inline package.json:40]: Sticky the version of the package 🙌🏼

File: src/components/atoms/Select/index.tsx [hunk 2]
```diff
@@ -1,61 +1,58 @@
...
+      classNamePrefix="select"
+      className={`select ${className}`}
+      aria-label={ariaLabel}
+      placeholder={placeholder}
+      isSearchable={isSearchable}
+      isClearable={isClearable}
+      options={items}
+      value={value}
+      onChange={value => onValueChange(value)}
+      noOptionsMessage={() => "No options"}
```
Comment (valentinoConti) [inline src/components/atoms/Select/index.tsx:44]: This is preference but I think "No results found" or "No options found" would be a better message 💪🏼 (asking George or Sofi for the wording would be better tho 😄)

Comment (raop155) [reply src/components/atoms/Select/index.tsx:44]: Asking and fixing it!

File: src/components/atoms/Select/styles.scss [hunk 3]
```diff
@@ -49,17 +63,29 @@
...
       text-align: center;
     }
+
+    &::-webkit-scrollbar {
+      width: 12px;
+      height: 4px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+      border: 4px solid rgba(0, 0, 0, 0);
```
[the remark this answers is not in the document]
Comment (raop155) [reply src/components/atoms/Select/styles.scss:73]: I think it is!

**Pull request merged**